README.md

```diff
@@ -54,3 +84,163 @@ To build the pds-server RPM package
 1. Checkout the project repo on a host of the OS type you would like to build the package for and change to that directory
 3. Run `make clean`
 4. Run `make package`
+
```

Comment:
We don't need to repeat the pds-cli docs, they are in the `docs/` subfolder of the `pds-cli` folder.